lib/logstash/inputs/kafka.rb

```diff
@@ -340,6 +342,24 @@ def create_consumer(client_id)
     end
   end
 
+  def partition_assignment_strategy_class(partition_assignment_strategy = self.partition_assignment_strategy)
```

Comment:
Does it make sense to put the opt param when the method is always invoked without parameter?

Reply:
thx, have thought about potentially unit testing the method - that's the original reasoning.
going to refactor (there's another very similar one in output) to not receive any arguments.